frontend/src/app/modules/home/dashboard/dashboard.component.ts

```diff
@@ -207,4 +214,51 @@ export class DashboardComponent extends BaseComponent implements OnInit, OnDestr
                 this.toastNotificationService.error(error);
             });
     }
+
+    saveTilesOrder() {
+        this.spinnerService.show(true);
+
+        this.tiles = this.tiles.map((t, i) => ({ ...t, tileOrder: i + 1 }));
```

Comment:
It would be better to reload ordered tiles after saving

Reply:
Fixed